x/evidence/spec/08_app_wiring.md

```diff
@@ -0,0 +1,5 @@
+# App Wiring
```

Comment:
will this be expanded? not sure it needs its own file, not entirely sure why it was added to the docs as well?

Reply:
I actually added it in every PR, I thought it might help if you want to have an app with only specifics modules. You tell me if not needed, we can use this PR to remove from everywhere.

Reply:
I could be wrong, but this is related to how testing will be done which will have its own documentation. Right now the spec describes module specific designs. 

Would a user need to look at this file? trying to better understand when someone would look at it